test/inductor/test_torchinductor.py

```diff
@@ -7543,6 +7544,7 @@ def fn(x: torch.Tensor) -> torch.Tensor:
                     fn_opt(inps)
 
         @skipIfRocm
+        @skip_if_consumer_card
```

Comment:
The test works for me on a 2060. What's issues are you seeing on a 3090?

Reply:
Subprocess is failing with:
```
b'Error: mkl-service + Intel(R) MKL: MKL_THREADING_LAYER=INTEL is incompatible with libgomp.so.1 library.\n\tTry to import numpy first or set the threading layer accordingly. Set MKL_SERVICE_FORCE_INTEL to force it.\n'
```

Actually might not be related to consumer cards, that was a guess based on test names.  I think I can fix it with `MKL_THREADING_LAYER=GNU`.

Reply:
That looks like #37377 which should be fixed by updating `mkl-service`. Your workaround seems fine though.